test/month_calendar/index.js

```diff
@@ -0,0 +1,44 @@
+const { createElement } = require('react');
+const { assert, expect, spy } = require('chai');
+const { mount } = require('enzyme');
+
+const locale = require('./locale');
+const theme = require('./theme');
+const MonthCalendar = require('../../lib/month_calendar');
+
+describe('MonthCalendar', () => {
```

Comment:
I think testing 'select a date' (below) and 'go back two months, check if 13th is at the right place' are the most important features to test. With those two users users are always able to select the date they need to enter. Opinion?

Others are:
- Cannot select dates before minDate
- Cannot select dates after maxDate